src/shared/api/api.js

```diff
@@ -89,25 +89,27 @@ function graphql({
       query,
       variables,
     }),
-  })
-    .then(async (res) =>
-      res.ok
-        ? await res.json()
-        : Promise.reject({
-            status: res.status,
-            data: await res.json(),
-          })
-    )
-    .catch((err) => {
-      if (err.status === 401 && config.IS_SELF_HOSTED) {
+  }).then(async (res) => {
+    const data = await res.json()
```

Comment:
`h`: What happens if the call to `res.json()` throws? I believe that was also being handled by the catch.

Reply:
fair enough, we used to have: if res.ok => res.json(), otherwise throw an error, this was recently changed when i added the catch block.